### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by Pacane**
After reading your code for a while, I think I could find some improvements.

I'm pretty sure everyone agrees with me on these changes. Right?

### CODE REVIEW

**Comment by jipiboily** [comment]
Ah...ah...ah... :P

For anyone coming here, I presented that project at a local meetup. I am French Canadian (as everyone is at that meetup). I thought that "dix-patch" would be a good name, as in french it would be pronounced "dispatch"...but as you can imagine, someone in the assistance made me realize how it would be pronounced in English...

We laughed very hard at my first slide (a 5 minute prep slide deck), as it's the worst slide ever, for various reasons, including the name.

Won't merge buddy, but...thanks....?!? :stuck_out_tongue:

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ You can send events to Forwardlytics, and it will forward it to many services li
```
Comment (GabLeRoux) [inline README.md]: You forgot the first line of the readme

Comment (Pacane) [inline README.md]: Feel free to contribute.

**Pull request closed without merging**